tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "compilerOptions": {
```

Comment:
Missing `strict: true` maybe? Or does that not work for some reason? I see it's explicitly `false` in [`@tsconfig/svelte`](https://www.npmjs.com/package/@tsconfig/svelte)

Reply:
Looks like `strict: true` is fine: https://github.com/sveltejs/kit/issues/4395

Will add.